src/Events/Integrations/Modules/Ticket_Emails/Hooks.php

```diff
@@ -0,0 +1,166 @@
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$this->add_actions();
+		$this->add_filters();
+	}
+
+	/**
+	 * Adds the actions required by each Tickets Emails component.
+	 *
+	 * @since TBD
+	 */
+	protected function add_actions() {
+		add_action( 'tribe_template_before_include:tickets/v2/emails/template-parts/body/tickets', [ $this, 'include_event_date_ticket_rsvp_emails' ], 10, 3 );
+		add_action( 'tribe_template_before_include:tickets/v2/emails/template-parts/body/tickets', [ $this, 'include_event_title_ticket_rsvp_emails' ], 10, 3 );
+		add_action( 'tribe_template_before_include:tickets/v2/emails/template-parts/body/tickets', [ $this, 'include_event_image_ticket_rsvp_emails' ], 10, 3 );
+		add_action( 'tribe_template_after_include:tickets/v2/emails/template-parts/body/tickets', [ $this, 'include_event_venue_ticket_rsvp_emails' ], 10, 3 );
+		add_action( 'tribe_template_after_include:tickets/v2/emails/template-parts/body/tickets', [ $this, 'maybe_include_event_links_ticket_rsvp_emails' ], 10, 3 );
+	}
+
+	/**
+	 * Adds the filters required by each Tickets Emails component.
+	 *
+	 * @since TBD
+	 */
+	protected function add_filters() {
+		// General emails filters.
+		add_filter( 'tec_tickets_emails_placeholders', tribe_callback( Emails::class, 'filter_tec_tickets_emails_placeholders' ), 10, 3 );
+		add_filter( 'tec_tickets_emails_preview_args', tribe_callback( Emails::class, 'filter_tec_tickets_emails_preview_args' ), 10, 4 );
+		add_filter( 'tec_tickets_emails_template_args', tribe_callback( Emails::class, 'filter_tec_tickets_emails_template_args' ), 10, 4 );
+
+		// Ticket Email.
+		add_filter( 'tec_tickets_emails_ticket_settings', tribe_callback( \TEC\Events\Integrations\Modules\Ticket_Emails\Email\Ticket::class, 'filter_tec_tickets_emails_ticket_email_settings' ), 10 );
+		add_filter( 'tec_tickets_emails_ticket_attachments', tribe_callback( \TEC\Events\Integrations\Modules\Ticket_Emails\Email\Ticket::class, 'filter_tec_tickets_emails_ticket_email_attachments' ), 10, 3 );
+
+		// RSVP Email.
+		add_filter( 'tec_tickets_emails_rsvp_settings', tribe_callback( \TEC\Events\Integrations\Modules\Ticket_Emails\Email\RSVP::class, 'filter_tec_tickets_emails_rsvp_email_settings' ), 10 );
+		add_filter( 'tec_tickets_emails_rsvp_attachments', tribe_callback( \TEC\Events\Integrations\Modules\Ticket_Emails\Email\RSVP::class, 'filter_tec_tickets_emails_rsvp_email_attachments' ), 10, 3 );
+	}
+
+	/**
+	 * Include the Event date in the ticket and RSVP emails.
+	 *
+	 * @since TBD
+	 *
+	 * @param string           $file        Template file.
+	 * @param string           $name        Template name.
+	 * @param \Tribe__Template $et_template Event Tickets template object.
+	 * @return void
+	 */
+	public function include_event_date_ticket_rsvp_emails( $file, $name, $et_template ) {
+		if ( ! $et_template instanceof \Tribe__Template ) {
+			return;
+		}
```

Comment:
I think these checks are redundant since the filter will _always_ pass a `Tribe__Template` [here](https://github.com/the-events-calendar/tribe-common/blob/master/src/Tribe/Template.php#L1420).

Reply:
It just enforces in case someone is using this elsewhere.